File: Maxglass.Ecommerce/Maxglass.Ecommerce.DataTransfer/Clientes/Responses/ClientePessoaFisicaResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Maxglass.Ecommerce.DataTransfer.Clientes.Responses
{
    public class ClientePessoaFisicaResponse : ClienteResponse
    {
        public string? Nome { get; set; }
        public string? SobreNome { get; set; }
        public string? Cpf { get; set; }
        public string? Telefone { get; set; }

    }
}

File: Maxglass.Ecommerce.Web/src/app/clientes/models/cliente-pessoa-fisica.response.ts
export class ClientePessoaFisicaResponse {
  email: string;
  cep: string;
  nome: string;
  sobreNome: string;
  cpf: string;
  telefone: string;

  constructor(params: Partial<ClientePessoaFisicaResponse>) {
    this.email = params.email || null;
    this.cep = params.cep || null;
    this.nome = params.nome || null;
    this.sobreNome = params.sobreNome || null;
    this.cpf = params.cpf || null;
    this.telefone = params.telefone || null;

  }
}
